ansible/roles/captain/vars/main.yml

```diff
@@ -1,5 +1,9 @@
 deploy_env: captain
 # supervisor_task_conf: "{{ service_home }}/supervisor"
+#
+# Defined in config directory:
+# softlayer_vpn_username: ...
+# softlayer_vpn_password: ...
```

Comment:
We should follow up offline on this. Practical security aside we should also understand the legal constraints (e.g. HIPAA) and make sure it's ok from that perspective.